hardware/tv-hat/README.md

```diff
@@ -0,0 +1,58 @@
+## Getting started with the Raspberry Pi TV HAT
+
+The TV HAT allows you to receive and decode digital television streams on your Raspberry Pi through its on-board DVB-T2 tuner. Then you can watch these streams on any computer connected to the same network as the Pi.
```

Comment:
The way this sentence is worded, it's not clear whether the TV HAT has a DVB-T2 tuner, or whether the Raspberry Pi has a DVB-T2 tuner which the TV HAT somehow enables :wink: 

I wonder if "The TV HAT has an on-board DVB-T2 tuner that allows you to receive and decode digital television streams on your Raspberry Pi." would be clearer?

Reply:
+1 for lurch's amendment, although this may well have been caught by the copy editors anyway.